### PR TITLE
[RFC] (Re)add a grey favicon for the backend

### PR DESCRIPTION
**Pull request by agoat**
I liked the backend icon. I want to (re)add this as it was removed in ad7b624943c4e9c6ce4d8995ef408b51717d264b.

Some people get confused to the same orange icon for the contao, community and backend pages (#958). So I made the icon grey to differ from the other pages.

### CODE REVIEW

File: src/Resources/contao/templates/backend/be_main.html5 [hunk 1]
```diff
@@ -9,6 +9,9 @@
   <meta name="viewport" content="width=device-width,initial-scale=1.0,shrink-to-fit=no">
   <meta name="referrer" content="origin">
 
+<?php $this->block('favicon'); ?>
+  <link rel="icon" type="image/png" href="<?= TL_ASSETS_URL ?>bundles/contaocore/favicon.ico">
+<?php $this->endblock(); ?>
```
Comment (fritzmg) [inline src/Resources/contao/templates/backend/be_main.html5]: This block should be indented by two spaces.

**Comment by leofeyer** [comment]
We have discussed this in Mumble yesterday and agreed that the back end should show the favicon that is in the document root.

**Comment by agoat** [comment]
> We have discussed this in Mumble yesterday and agreed that the back end should show the favicon that is in the document root.

Ah, sorry. I didn't know that.

**Comment by fritzmg** [comment]
@agoat you could create a PR to introduce template blocks into the back end templates instead. e.g.
```php
…
<head>

  <meta charset="<?= $this->charset ?>">
  <title><?= $this->title ?> - Contao Open Source CMS</title>
  <base href="<?= $this->base ?>">

  <?php $this->block('meta'); ?>
    <meta name="generator" content="Contao Open Source CMS">
    <meta name="viewport" content="width=device-width,initial-scale=1.0,shrink-to-fit=no">
    <meta name="referrer" content="origin">
  <?php $this->endblock(); ?>

  <?php $this->block('stylesheets'); ?>
    <link rel="stylesheet" href="<?= TL_ASSETS_URL ?>system/themes/<?= $this->theme ?>/fonts.css">
    <link rel="stylesheet" href="<?= TL_ASSETS_URL ?>assets/colorpicker/css/mooRainbow.min.css">
    <link rel="stylesheet" href="<?= TL_ASSETS_URL ?>assets/chosen/css/chosen.min.css">
    <link rel="stylesheet" href="<?= TL_ASSETS_URL ?>assets/simplemodal/css/simplemodal.min.css">
    <link rel="stylesheet" href="<?= TL_ASSETS_URL ?>assets/datepicker/css/datepicker.min.css">
    <link rel="stylesheet" href="<?= TL_ASSETS_URL ?>system/themes/<?= $this->theme ?>/basic.css">
    <link rel="stylesheet" href="<?= TL_ASSETS_URL ?>system/themes/<?= $this->theme ?>/main.css">
    <?= $this->stylesheets ?>
  <?php $this->endblock(); ?>

  <?php $this->block('javascripts'); ?>
    <script><?= $this->getLocaleString() ?></script>
    <script src="<?= TL_ASSETS_URL ?>assets/mootools/js/mootools.min.js"></script>
    <script src="<?= TL_ASSETS_URL ?>assets/colorpicker/js/mooRainbow.min.js"></script>
    <script src="<?= TL_ASSETS_URL ?>assets/chosen/js/chosen.min.js"></script>
    <script src="<?= TL_ASSETS_URL ?>assets/simplemodal/js/simplemodal.min.js"></script>
    <script src="<?= TL_ASSETS_URL ?>assets/datepicker/js/datepicker.min.js"></script>
    <script src="<?= TL_ASSETS_URL ?>bundles/contaocore/mootao.min.js"></script>
    <script src="<?= TL_ASSETS_URL ?>bundles/contaocore/core.min.js"></script>
    <script src="<?= TL_ASSETS_URL ?>system/themes/<?= $this->theme ?>/hover.js"></script>
    <script><?= $this->getDateString() ?></script>
    <?= $this->javascripts ?>
  <?php $this->endblock(); ?>

</head>
…
```
This way you could create your own `be_main.html5` template and do something like
```php
<?php $this->extend('be_main'); ?>

<?php $this->block('stylesheets'); ?>
  <link rel="icon" type="image/png" href="<?= TL_ASSETS_URL ?>path/to/your/backend/favicon.ico">
  <?php $this->parent(); ?>
<?php $this->endblock(); ?>
```
for example, if you still want to have a custom backend favicon.

**Comment by agoat** [comment]
> @agoat you could create a PR to introduce template blocks into the back end templates instead. e.g.

I think I will go with the document root solution for now and copy my grey icon there.

**Pull request closed without merging**